dataland-frontend/tests/e2e/specs/data-request/DataRequestProcess.ts

```diff
@@ -166,7 +166,7 @@ describe("As a user I expect a data request page where I can download an excel t
         [
           {
             filename: tooLargeFilename,
-            fileSize: DATA_REQUEST_UPLOAD_MAX_FILE_SIZE_IN_BYTES + 1,
+            fileSize: parseInt(getStringCypressEnv("DATA_REQUEST_UPLOAD_MAX_FILE_SIZE_IN_BYTES")) + 1,
```

Comment:
Why is the env no number in the first place?

Reply:
it seems that cypress envs are always strings per default.  this is why the value needs to be casted to a number here first